ant-cli/src/actions/download.rs

```diff
@@ -127,3 +142,51 @@ async fn download_public(
         Err(eyre!("Errors while downloading data"))
     }
 }
+
+async fn download_public_single_file(
+    addr: &str,
+    address: DataAddr,
+    dest_path: &str,
+    client: &Client,
+) -> Result<()> {
+    let bytes = match client.data_get_public(&address).await {
+        Ok(bytes) => bytes,
+        Err(e) => {
+            let err = format!("Failed to fetch file at {addr:?}: {e}");
+            return Err(eyre!(err)).wrap_err("Failed to fetch file content from address");
```

Comment:
better to include the address in the error msg

Reply:
It's included line 155 no?